tools/projmgr/src/ProjMgrGenerator.cpp

```diff
@@ -219,6 +219,21 @@ void ProjMgrGenerator::GenerateCprjVector(XMLTreeElement* element, const vector<
   }
 }
 
+void ProjMgrGenerator::GenerateCprjOptions(XMLTreeElement* element, const BuildType& buildType) {
+
+  if (buildType.optimize.empty() && buildType.debug.empty() && buildType.warnings.empty()) {
+    return;
+  }
+
+  XMLTreeElement *optionsElement = element->CreateElement("options");
+  if (optionsElement)
+  {
+    SetAttribute(optionsElement, "optimize", buildType.optimize);
+    SetAttribute(optionsElement, "debug", buildType.debug);
+    SetAttribute(optionsElement, "warnings", buildType.warnings);
+  }
```

Comment:
@tarek-bochkati Codecov fails due to decreased code coverage in this PR. Could you please add a test case to cover lines 228-234? Thanks!

Reply:
@brondani I have added a test that I expect it to cover this code chunk.